user.go

```diff
@@ -35,16 +36,17 @@ type Bank struct {
 
 // UserProfile represents a user's personal and financial profile.
 type UserProfile struct {
-	UserName      string   `json:"user_name"`
-	UserShortName string   `json:"user_shortname"`
-	AvatarURL     string   `json:"avatar_url"`
-	UserType      string   `json:"user_type"`
-	Email         string   `json:"email"`
-	Phone         string   `json:"phone"`
-	Broker        string   `json:"broker"`
-	Products      []string `json:"products"`
-	OrderTypes    []string `json:"order_types"`
-	Exchanges     []string `json:"exchanges"`
+	UserName      string           `json:"user_name"`
+	UserShortName string           `json:"user_shortname"`
+	AvatarURL     string           `json:"avatar_url"`
+	UserType      string           `json:"user_type"`
+	Email         string           `json:"email"`
+	Phone         string           `json:"phone"`
+	Broker        string           `json:"broker"`
+	Meta          *json.RawMessage `json:"meta"`
```

Comment:
Similar to the previous comment, is there a defined format for Meta?

Reply:
This is updated to a struct.